src/EnergyPlus/VentilatedSlab.cc

```diff
@@ -774,13 +774,13 @@ namespace VentilatedSlab {
                                                        DataLoopNode::NodeFluidType::Air,
                                                        DataLoopNode::ConnectionType::Inlet,
                                                        NodeInputManager::CompFluidStream::Primary,
-                                                       ObjectIsNotParent);
+                                                       ObjectIsParent);
 
                 ventSlab.OAMixerOutNode = GetOnlySingleNode(state,
                                                             state.dataIPShortCut->cAlphaArgs(23),
                                                             ErrorsFound,
                                                             DataLoopNode::ConnectionObjectType::ZoneHVACVentilatedSlab,
-                                                            ventSlab.Name + "-OA MIXER",
+                                                            ventSlab.Name,
```

Comment:
These GetOnlySingleNode call arguments did not allow the child objects to be associated with the parent object. For this reason, the ventilated slab could not find child components.

Reply:
> These GetOnlySingleNode call arguments did not allow the child objects to be associated with the parent object. For this reason, the ventilated slab could not find child components.

Hmm, but there was a reason for adding the suffix. Does HVAC Diagram still draw for this configuration? Aren't there compset calls to establish parent/child reationships?

Reply:
No it does not draw and there are no node connection errors in the err file. I guess there is more work needed here to get both svg and parent/child relationship working.

Reply:
Related discussion to node connections for ventilated slab. https://github.com/NREL/EnergyPlus/issues/2682

Reply:
This is just strange. In release, the svg appears correct (OA nodes at bottom of drawing are not connected) but it can't find the child objects (err file shows new error below and html shows "Unknown"). In debug, it finds the child objects but can't draw the svg correctly (and err file is clean and html shows parent/child relation in tables). These runs are with the original VentilatedSlab.cc file with all changes to that file reverted (actually copied from develop). For a little clue, in the release svg drawing the top 3 are SlabAndZone and bottom 2 are SlabOnly. In the debug svg, all vent slabs show at the bottom but not at the top with the internal OA mixer connection. Something is wrong with these node registrations but this debug vs release difference throws a wrench in things.

    ** Warning ** Parent object not found for zone coil = ZONE1VENTSLABCOOLINGCOIL

Debug:
![image](https://github.com/NREL/EnergyPlus/assets/5778156/401d3c3d-37b8-4893-95d7-6abdf1e83e97)

Release:
![image](https://github.com/NREL/EnergyPlus/assets/5778156/e2e50832-b83a-463b-8afb-5323af62a38e)

Reply:
Hmm, will look at the svg outputs later.